### ISSUE TITLE
Error when including stdlib.h Clang

### ISSUE THREAD

**Issue by LukePrior** [taybenlor/runno#193]
The program will fail to compile if `stdlib.h` is included.

Three errors occur all appearing to do with `wchar_t`

```
In file included from program:2:
/sys/include/stdlib.h:83:13: error: unknown type name 'wchar_t'
int mbtowc (wchar_t *__restrict, const char *__restrict, size_t);
            ^
/sys/include/stdlib.h:85:18: error: unknown type name 'wchar_t'
size_t mbstowcs (wchar_t *__restrict, const char *__restrict, size_t);
                 ^
/sys/include/stdlib.h:86:42: error: unknown type name 'wchar_t'
size_t wcstombs (char *__restrict, const wchar_t *__restrict, size_t);
```

Example [here](https://runno.run/?editor=1&runtime=clang&code=I2luY2x1ZGUgPHN0ZGxpYi5oPgoKaW50IG1haW4odm9pZCkgewogICAgCiAgICByZXR1cm4gMDsKfQo.).

**Comment by LukePrior** [comment]
I tried compiling the code using [wasm-clang](https://binji.github.io/wasm-clang/) directly and it seems to have worked so I'm not sure what the issue is.

![image](https://user-images.githubusercontent.com/22492406/162152491-cb30cc51-e41e-4202-ad1c-66270b5f6f8e.png)

**Comment by taybenlor** [comment]
If you switch to the `clangpp` runtime this seems to work fine. The `clang` available on WAPM (https://wapm.io/syrusakbary/clang) had a bunch of issues with C++ so I fixed them in `runno-clang` (https://wapm.io/taybenlor/runno-clang) by copying `wasm-clang`. This could probably be fixed by switching uses of `clang` in the C runtime to `runno-clang` (PR welcome!).

**Comment by taybenlor** [comment]
This is fixed by the new runtime in `v0.4.2`
